.github/workflows/test.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
+on: [push]
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install
+      - uses: borales/actions-yarn@v2.3.0
```

Comment:
What is this image? Is it official?

Reply:
it's a wrapper of the official package (this: https://github.com/actions/setup-node) to work with yarn. we can easily use the default one like this:
```
steps:
- uses: actions/checkout@v2
- uses: actions/setup-node@v2
  with:
    node-version: '14'
- run: yarn install
- run: yarn test
```

but from my testing this one runs faster